README.rst

```diff
@@ -38,8 +38,7 @@ The expected information in run_info.xml is::
       <startDateTime date="2020-01-26" time="10:00:00"/>
       <endDateTime date="2020-01-30" time="12:00:00"/>
       <properties>
-          <string key="username" value="pietje"/>
-          <string key="password" value="onder-de-deurmat"/>
+          <string key="api_token" value="token"/>
```

Comment:
Beste is om de vorm van het echte token te gebruiken. Dus zoiets als `aBCd1234.5678tOkeNabcde`, dat helpt in de praktijk met herkennen. "API token zal elders "api key" of "personal access token" genoemd worden, dat wordt vaak niet in sync gehouden. De vorm van random strings met een punt ertussen, dat helpt met het herkennen dat je de goeie pakt.

Reply:
Oké goede en ik heb het aangepast. Binnen threedi heet het personal api token, vandaar dat ik api token heb genoemd.